src/black/__init__.py

```diff
@@ -196,6 +206,14 @@ def validate_regex(
         " when piping source on standard input)."
     ),
 )
+@click.option(
+    "--ipynb",
+    is_flag=True,
+    help=(
+        "Format all input files like ipynb notebooks regardless of file extension "
+        "(useful when piping source on standard input)."
+    ),
+)
```

Comment:
I'm not the biggest fan of this since this allows for "illegal state" to be configured (forcing all files to be treated both as stubs and jupyter notebooks). While this doesn't break anything (the --ipynb flag wins out), it's probably not the best UX. Totally down to deal with this in a later PR (although before this gets released). Also, totally fine to leave this alone, it's not that terrible anyway.

Reply:
Yeah, that's a good point...I'll remove it for now then